internal/types/reusable.go

```diff
@@ -27,17 +27,17 @@ import (
 
 	"github.com/olekukonko/tablewriter"
 	"github.com/spf13/cobra"
-	"gopkg.in/yaml.v3"
-
 	"github.com/tj-actions/auto-doc/internal"
 	"github.com/tj-actions/auto-doc/internal/utils"
+	"gopkg.in/yaml.v3"
```

Comment:
Since this a third-party package we can revert this change

Reply:
Sure, reverted.

Reply:
Actually this appears to be automated

Reply:
Yes, exactly.